Binary files i686-unknown-linux/lib/arg.o and i386-unknown-bsdi3.1/lib/arg.o differ
Binary files i686-unknown-linux/lib/array.o and i386-unknown-bsdi3.1/lib/array.o differ
Binary files i686-unknown-linux/lib/assert.o and i386-unknown-bsdi3.1/lib/assert.o differ
Binary files i686-unknown-linux/lib/bitvec.o and i386-unknown-bsdi3.1/lib/bitvec.o differ
Binary files i686-unknown-linux/lib/buffer.o and i386-unknown-bsdi3.1/lib/buffer.o differ
Binary files i686-unknown-linux/lib/core.o and i386-unknown-bsdi3.1/lib/core.o differ
Binary files i686-unknown-linux/lib/dict.o and i386-unknown-bsdi3.1/lib/dict.o differ
Binary files i686-unknown-linux/lib/dirent.o and i386-unknown-bsdi3.1/lib/dirent.o differ
Binary files i686-unknown-linux/lib/execinfo.o and i386-unknown-bsdi3.1/lib/execinfo.o differ
Binary files i686-unknown-linux/lib/fcntl.o and i386-unknown-bsdi3.1/lib/fcntl.o differ
Binary files i686-unknown-linux/lib/filename.o and i386-unknown-bsdi3.1/lib/filename.o differ
Binary files i686-unknown-linux/lib/fn.o and i386-unknown-bsdi3.1/lib/fn.o differ
Binary files i686-unknown-linux/lib/getopt.o and i386-unknown-bsdi3.1/lib/getopt.o differ
Binary files i686-unknown-linux/lib/getopt1.o and i386-unknown-bsdi3.1/lib/getopt1.o differ
Binary files i686-unknown-linux/lib/grp.o and i386-unknown-bsdi3.1/lib/grp.o differ
Binary files i686-unknown-linux/lib/hashtable.o and i386-unknown-bsdi3.1/lib/hashtable.o differ
Binary files i686-unknown-linux/lib/in.o and i386-unknown-bsdi3.1/lib/in.o differ
Binary files i686-unknown-linux/lib/inet.o and i386-unknown-bsdi3.1/lib/inet.o differ
Binary files i686-unknown-linux/lib/lexing.o and i386-unknown-bsdi3.1/lib/lexing.o differ
Binary files i686-unknown-linux/lib/libcyc.a and i386-unknown-bsdi3.1/lib/libcyc.a differ
Binary files i686-unknown-linux/lib/lineno.o and i386-unknown-bsdi3.1/lib/lineno.o differ
Binary files i686-unknown-linux/lib/list.o and i386-unknown-bsdi3.1/lib/list.o differ
Binary files i686-unknown-linux/lib/mman.o and i386-unknown-bsdi3.1/lib/mman.o differ
Binary files i686-unknown-linux/lib/netdb.o and i386-unknown-bsdi3.1/lib/netdb.o differ
Binary files i686-unknown-linux/lib/nogc.a and i386-unknown-bsdi3.1/lib/nogc.a differ
Binary files i686-unknown-linux/lib/nogc.o and i386-unknown-bsdi3.1/lib/nogc.o differ
Binary files i686-unknown-linux/lib/position.o and i386-unknown-bsdi3.1/lib/position.o differ
Binary files i686-unknown-linux/lib/pp.o and i386-unknown-bsdi3.1/lib/pp.o differ
Binary files i686-unknown-linux/lib/printf.o and i386-unknown-bsdi3.1/lib/printf.o differ
Binary files i686-unknown-linux/lib/pwd.o and i386-unknown-bsdi3.1/lib/pwd.o differ
Binary files i686-unknown-linux/lib/queue.o and i386-unknown-bsdi3.1/lib/queue.o differ
Binary files i686-unknown-linux/lib/ref.o and i386-unknown-bsdi3.1/lib/ref.o differ
Binary files i686-unknown-linux/lib/rope.o and i386-unknown-bsdi3.1/lib/rope.o differ
Binary files i686-unknown-linux/lib/runtime_cyc.o and i386-unknown-bsdi3.1/lib/runtime_cyc.o differ
Binary files i686-unknown-linux/lib/scanf.o and i386-unknown-bsdi3.1/lib/scanf.o differ
Binary files i686-unknown-linux/lib/set.o and i386-unknown-bsdi3.1/lib/set.o differ
Binary files i686-unknown-linux/lib/signal.o and i386-unknown-bsdi3.1/lib/signal.o differ
Binary files i686-unknown-linux/lib/slowdict.o and i386-unknown-bsdi3.1/lib/slowdict.o differ
Binary files i686-unknown-linux/lib/socket.o and i386-unknown-bsdi3.1/lib/socket.o differ
Binary files i686-unknown-linux/lib/splay.o and i386-unknown-bsdi3.1/lib/splay.o differ
Binary files i686-unknown-linux/lib/stat.o and i386-unknown-bsdi3.1/lib/stat.o differ
Binary files i686-unknown-linux/lib/stdio.o and i386-unknown-bsdi3.1/lib/stdio.o differ
Binary files i686-unknown-linux/lib/stdlib.o and i386-unknown-bsdi3.1/lib/stdlib.o differ
Binary files i686-unknown-linux/lib/string.o and i386-unknown-bsdi3.1/lib/string.o differ
Binary files i686-unknown-linux/lib/time.o and i386-unknown-bsdi3.1/lib/time.o differ
Binary files i686-unknown-linux/lib/unistd.o and i386-unknown-bsdi3.1/lib/unistd.o differ
Binary files i686-unknown-linux/lib/xarray.o and i386-unknown-bsdi3.1/lib/xarray.o differ
Binary files i686-unknown-linux/src/absyn.o and i386-unknown-bsdi3.1/src/absyn.o differ
Binary files i686-unknown-linux/src/absyndump.o and i386-unknown-bsdi3.1/src/absyndump.o differ
Binary files i686-unknown-linux/src/absynpp.o and i386-unknown-bsdi3.1/src/absynpp.o differ
Binary files i686-unknown-linux/src/cf_absexp.o and i386-unknown-bsdi3.1/src/cf_absexp.o differ
Binary files i686-unknown-linux/src/cf_flowinfo.o and i386-unknown-bsdi3.1/src/cf_flowinfo.o differ
Binary files i686-unknown-linux/src/cyclone and i386-unknown-bsdi3.1/src/cyclone differ
Binary files i686-unknown-linux/src/cyclone.o and i386-unknown-bsdi3.1/src/cyclone.o differ
Binary files i686-unknown-linux/src/evexp.o and i386-unknown-bsdi3.1/src/evexp.o differ
Binary files i686-unknown-linux/src/formatstr.o and i386-unknown-bsdi3.1/src/formatstr.o differ
diff -rbBcN --exclude=CVS i686-unknown-linux/src/install_path.c i386-unknown-bsdi3.1/src/install_path.c
*** i686-unknown-linux/src/install_path.c	Wed Dec 31 16:00:00 1969
--- i386-unknown-bsdi3.1/src/install_path.c	Wed Sep 12 19:34:53 2001
***************
*** 0 ****
--- 1,3 ----
+ char *Cdef_inc_path = "/usr/include/cyclone";
+ char *Carch = "i386-unknown-bsdi3.1";
+ char *Cdef_lib_path = "/usr/local/lib/cyclone";
Binary files i686-unknown-linux/src/install_path.o and i386-unknown-bsdi3.1/src/install_path.o differ
Binary files i686-unknown-linux/src/interface.o and i386-unknown-bsdi3.1/src/interface.o differ
Binary files i686-unknown-linux/src/lex.o and i386-unknown-bsdi3.1/src/lex.o differ
Binary files i686-unknown-linux/src/new_control_flow.o and i386-unknown-bsdi3.1/src/new_control_flow.o differ
Binary files i686-unknown-linux/src/parse_tab.o and i386-unknown-bsdi3.1/src/parse_tab.o differ
Binary files i686-unknown-linux/src/tc.o and i386-unknown-bsdi3.1/src/tc.o differ
Binary files i686-unknown-linux/src/tcdecl.o and i386-unknown-bsdi3.1/src/tcdecl.o differ
Binary files i686-unknown-linux/src/tcenv.o and i386-unknown-bsdi3.1/src/tcenv.o differ
Binary files i686-unknown-linux/src/tcexp.o and i386-unknown-bsdi3.1/src/tcexp.o differ
Binary files i686-unknown-linux/src/tcpat.o and i386-unknown-bsdi3.1/src/tcpat.o differ
Binary files i686-unknown-linux/src/tcstmt.o and i386-unknown-bsdi3.1/src/tcstmt.o differ
Binary files i686-unknown-linux/src/tcutil.o and i386-unknown-bsdi3.1/src/tcutil.o differ
Binary files i686-unknown-linux/src/toc.o and i386-unknown-bsdi3.1/src/toc.o differ
Binary files i686-unknown-linux/src/tovc.o and i386-unknown-bsdi3.1/src/tovc.o differ
